ee/clickhouse/sql/cohort.py

```diff
@@ -97,7 +83,7 @@
 SELECT person_id
```

Comment:
Also, it might be worth putting a `distinct person_id` in the select. Definitely a super edge case, but I'm wondering if there's a case where the select happens in the middle of the insert, and because inserts aren't atomic, we could end up with multiple copies of the same user.